src/richie/apps/courses/templates/courses/cms/course_detail.html

```diff
@@ -180,6 +180,13 @@ <h1 class="subheader__title" property="name">{% render_model current_page "title
                                 {% if open_visible_runs|length == 0 %}
                                     <div class="course-detail__row course-detail__runs course-detail__runs--open">
                                         <div class="course-detail__empty">{% trans "No open course runs" %}</div>
+                                        {% is_feature_enabled 'JOANIE_WISHLIST' as joanie_wishlist_enabled %}
+                                        {% is_joanie_enabled as is_joanie_enabled_bool %}
+                                        {% if is_joanie_enabled_bool and joanie_wishlist_enabled  %}
```

Comment:
Doesn't it work ?
```suggestion
                                        {% if is_joanie_enabled and joanie_wishlist_enabled  %}
```

Reply:
I just re-tested it and it didn't work : 
```python
{% if is_joanie_enabled %} BELLA {% else %} BILBO {% endif %}
``` 
[KO] display `BILBO`
and 
```python
{% is_joanie_enabled as is_joanie_enabled_bool %}
{% if is_joanie_enabled_bool %} HELLO {% else %} BYE BYE {% endif %}
```
[OK] display `HELLO`